programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,9 @@
+function solution(seoul) {
+  var answer = "";
+  for (let i in seoul) {
+    if (seoul[i] == "Kim") {
```

Comment:
```suggestion
    if (seoul[i] === "Kim") {
```
== 보다 ===을 사용하는 것을 권장합니다.
javascript에서 == 연산은 자료형이 달라도 true로 인식해요

ex ) 100 == '100' // true

===은 == 보다 엄격하게 비교해주는 것이라고 생각하시면 편해요.
===은  자료형까지 일치해야 true를 반환합니다.